src/network/netdev/tunnel.c

```diff
@@ -617,6 +629,33 @@ int config_parse_encap_limit(const char* unit,
         return 0;
 }
 
+CONFIG_PARSER_PROTOTYPE(config_parse_6rd_prefix) {
```

Comment:
I am not sure, but we have never used the macro in the definition...

Reply:
Oh, sorry! I misread the source file for the original comment. I changed the definition back.